source/states/PlayerGroundState.hx

```diff
@@ -0,0 +1,108 @@
+package states;
+
+import flixel.FlxG;
+import flixel.FlxObject;
+import flixel.input.keyboard.FlxKey;
+
+class PlayerGroundState extends BaseState
+{
+  public function new()
+  {
+    super();
+  }
+
+  private override function transition(object:FlxObject):Bool
+  {
+    var player = cast(object, Player);
+    var attemptedJump:Bool = player.isJumping();
+
+    if (attemptedJump)
+    {
+      nextState = new PlayerJumpState();
+      return true;
+    }
+
```

Comment:
Also need to check if the player isn't in contact with the ground, put them in the appropriate air state. Probably do an else if () to put them in fall state if they aren't jumping.